src/main/java/com/gameList/gamelist/GameListController.java

```diff
@@ -53,7 +53,7 @@ public GameListController(GameListService theGameList) {
 	 * SETUP NEEDED:
 	 * Note: it has the same admin and password we used in class
 	 * 1. On phpMyAdmin you will have to make a database called "game_list" and a table called "gamelist"
-	 * 2. this table will have 4 columns id(primary key + auto increment), gamename, console and year
+	 * 2. this table will have 5 columns id(primary key + auto increment), gamename, console, year and description
```

Comment:
I just added to your DB set-up instructions here because the 'description' field in GameList.java is mapped to a column, so it's needed for proper DB set-up. Will give an error if you try to run the application without a description column in the DB table.